source/Galaxy.cpp

```diff
@@ -45,3 +47,8 @@ const Sprite *Galaxy::GetSprite() const
 {
 	return sprite;
 }
+
+float Galaxy::Scale() const
+{
+	return scale;
+}
```

Comment:
Nit: Missing newline. Your text editor may not support EditorConfig definitions, as it should have ensured a newline at EOF.